### ISSUE TITLE
node6 tests on PRs fail on CI

### ISSUE THREAD

**Issue by kinwa91** [googleapis/nodejs-common#213]
both #210 and #212 are seeing test failures for node6 - node8/10 are fine.

Also, when I try to run tests locally:

```
nodejs-common git:(service-object-parent) npm run test-no-cover
npm WARN npm npm does not support Node.js v10.8.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/

> @google-cloud/common@0.21.1 test-no-cover /Users/jonathanlui/nodejs-common
> mocha build/test

Error: Cannot find module 'shimmer'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
```

**Comment by JustinBeckwith** [comment]
Confirmed, this totally "works on my machine" ™️

**Comment by kinwa91** [comment]
is #214 meant to fix the node6 issue?

**Comment by JustinBeckwith** [comment]
Yes and no :) #214 fixes the node 6 issue you have over in #212.  I don't believe there's an issue with `shimmer`, since it was only used in the logs.  Did running `gts clean` fix your issue?

**Comment by kinwa91** [comment]
Okay i see those are two separate issues, I will try again on my side.
